platform/feature-api/communication/postoffice/src/protocol.ts

```diff
@@ -23,15 +23,15 @@ export interface EndpointRequestPart {
 /**
  * A request comprising a chain of function calls.
  */
-export type EndpointRequest = EndpointRequestPart[];
+export type BackendRequest = BackendRequestPart[];
 
 /**
  * A response representing the return value of a function call.
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type EndpointResponse = any;
+export type BackendResponse = any;
```

Comment:
In this case, it's probably useless to define this type. Again, not part of the renaming, but if we're eventually refactoring, this would be a prime candidate.

Reply:
yeah that's true, not sure why it was defined like that in the first place,  maybe just to give it a name and change it later on if needed ?

Reply:
You can also change if you just use `any`.

Reply:
that's true, but it is far easier to change a type that has already been implemented than putting any every where and then changing every any to the type of interest, if a need for a format arises for the responses. Again, it is odd to put a type and define it as any but I am trying to work out what they might've been thinking :)